api/src/privacy/domain/usecases/anonymize-user.usecase.js

```diff
@@ -71,6 +80,45 @@ const anonymizeUser = withTransaction(async function ({
   }
 });
 
+async function _anonymizeMembershipsLastAccessedAt(membershipRepository, userId) {
+  const userMemberships = await membershipRepository.findByUserId(userId);
+
+  for (const membership of userMemberships) {
+    const anonymizedMembershipLastAccessedAt = anonymizeGeneralizeDate(membership.lastAccessedAt);
+    await membershipRepository.updateLastAccessedAt({
+      membershipId: membership.id,
+      lastAccessedAt: anonymizedMembershipLastAccessedAt,
+    });
+  }
+}
```

Comment:
```suggestion
async function _anonymizeMemberships({ userId, anonymizedByUserId, membershipRepository }) {
  // anonymize last access at
  const userMemberships = await membershipRepository.findByUserId(userId);

  for (const membership of userMemberships) {
    const anonymizedMembershipLastAccessedAt = anonymizeGeneralizeDate(membership.lastAccessedAt);
    await membershipRepository.updateLastAccessedAt({
      membershipId: membership.id,
      lastAccessedAt: anonymizedMembershipLastAccessedAt,
    });
  }
  
  // Disable Memberships
  await membershipRepository.disableMembershipsByUserId({ userId, updatedByUserId: anonymizedByUserId });
}
```

Faire de même avec `_anonymizeCertificationCenterMembershipsLastAccessedAt`